mex/extractors/igs/transform.py

```diff
@@ -6,23 +6,36 @@
     ExtractedContactPoint,
     ExtractedPrimarySource,
     ExtractedResource,
+    ExtractedVariable,
+    ExtractedVariableGroup,
     ResourceMapping,
+    VariableMapping,
+)
+from mex.common.types import (
+    MergedOrganizationalUnitIdentifier,
+    MergedResourceIdentifier,
+    MergedVariableGroupIdentifier,
+    Text,
+)
+from mex.extractors.igs.model import (
+    IGSEnumSchema,
+    IGSInfo,
+    IGSPropertiesSchema,
+    IGSSchema,
 )
-from mex.common.types import MergedOrganizationalUnitIdentifier, Text, TextLanguage
-from mex.extractors.igs.model import IGSEnumSchema, IGSSchema
 
 
 def transform_igs_schemas_to_resources(
```

Comment:
```suggestion
def transform_igs_info_to_resources(
```
would it make sense to rename the function ten?

Reply:
https://github.com/robert-koch-institut/mex-extractors/pull/61/commits/de23aff08c43480f87328218e87329f0b1f4e544